src/types/pubsub.rs

```diff
@@ -70,6 +73,12 @@ pub enum PubsubMessage<P: Preset> {
     LightClientFinalityUpdate(Box<LightClientFinalityUpdate<P>>),
     /// Gossipsub message providing notification of a light client optimistic update.
     LightClientOptimisticUpdate(Box<LightClientOptimisticUpdate<P>>),
+    /// Gossipsub message providing notification of an execution payload envelope.
+    ExecutionPayload(Arc<SignedExecutionPayloadEnvelope<P>>),
+    /// Gossipsub message providing notification of a payload attestation message.
+    PayloadAttestationMessage(Arc<PayloadAttestationMessage>),
+    /// Gossipsub message providing notification of an execution payload bid.
+    ExecutionPayloadBid(Arc<SignedExecutionPayloadBid<P>>),
```

Comment:
```suggestion
    ExecutionPayloadBid(Arc<SignedExecutionPayloadBid>),
```
it doesn't depend on preset https://github.com/hangleang/grandine/blob/669b0b20e1b215a57ae4462f3c9ba1ec9dd2d39f/types/src/gloas/containers.rs#L226